src/disco/containers/DiscoPane.js

```diff
@@ -34,13 +34,17 @@ export class DiscoPaneBase extends React.Component {
     results: PropTypes.arrayOf(PropTypes.object).isRequired,
     _addChangeListeners: PropTypes.func,
     _tracking: PropTypes.object,
+    // We allow the video to be overloaded for testing, see:
+    // https://github.com/mozilla/addons-frontend/issues/765
```

Comment:
I don't think it's necessary to add a comment. This is a common pattern that we use elsewhere. The properties above it have defaults already too.